Sources/MongoSwift/Operations/ListCollectionsOperation.swift

```diff
@@ -99,7 +99,11 @@ internal struct ListCollectionsOperation: Operation {
                                                                 decoder: self.database.decoder,
                                                                 session: session,
                                                                 initializer: initializer)
-            return .names(cursor.map { $0["name"] as? String ?? "" })
+            return try .names(cursor.map { guard let name = $0["name"] as? String else {
+                throw RuntimeError.internalError(message: "Invalid server response: collection has no name")
```

Comment:
nit: the indentation is a little weird here in that the `throw` is lined up with the `}` in the line below. I think maybe we should indent this `throw...` line in one more.